modules/emulator/R/ldinvgamma.R

```diff
@@ -4,9 +4,9 @@
 ##' @title ldinvgamma
 ##' @export
 ##'
-##' @param x
-##' @param shape
-##' @param scale
+##' @param x Name of variable to plot on X axis
+##' @param shape shape of points
+##' @param scale scale, NULL by default
```

Comment:
A lot of the functions in this package that take `x` arguments aren't plotting anything, but rather computing densities of statistical distributions, so "name of variable to plot" isn't the right description. What to replace it with will differ from one function to another; here's a suggestion for this one, but please check the others too.

```suggestion
##' @param x vector of quantiles
##' @param shape,scale shape and scale parameters for the inverse Gamma distribution
```